docs/en/getting-started/index.asciidoc

```diff
@@ -11,11 +11,8 @@
 :blog-ref:          https://www.elastic.co/blog/
 :wikipedia:         https://en.wikipedia.org/wiki
 
-:kib-repo-dir:      {docdir}/../../../../kibana/docs
```

Comment:
Were these attributes intentionally removed?  I would expect them to just be updated with {stack-docs-root} paths

Reply:
I switched everything to `{kibana-root}/docs`. I didn't think we needed the extra indirection there. Is that ok?